src/DeviceInput/deviceInputSystem.ts

```diff
@@ -58,7 +58,12 @@ export class DeviceInputSystem {
      */
     constructor(deviceInputSystem: IDeviceInputSystem) {
         this._deviceInputSystem = deviceInputSystem;
-        this.onDeviceConnectedObservable = new Observable();
+        // Adds a callback that reassigns onDeviceConnected's callback to trigger the observable's observers for already connected devices
+        this.onDeviceConnectedObservable = new Observable((observer) => {
+            this._deviceInputSystem.onDeviceConnected = (deviceType, deviceSlot) => {
+                this.onDeviceConnectedObservable.notifyObservers({ deviceType, deviceSlot });
+            };

```

Comment:
I think I don't have enough context for how this is set up to know whether/how this fixes the input system breakage. This looks like it's assigning a callback inside an observable's callback, which looks like it might be called every time a device is connected and consequently pave over the old callback every time. Is that what's supposed to happen?

Reply:
Found an easier way to implement this fix